prime-router/src/test/kotlin/ElementTests.kt

```diff
@@ -49,10 +50,12 @@ internal class ElementTests {
             ),
             csvFields = Element.csvFields("b", format = "\$alt")
         )
-        val noResult = one.toNormalized("Non", "\$alt")
-        assertEquals("N", noResult)
-        val yesResult = one.toNormalized("Oui", "\$alt")
-        assertEquals("Y", yesResult)
+        one.toNormalized("Non", "\$alt").run {
+            assertThat(this).isEqualTo("N")
+        }
```

Comment:
Rather than create a `noResult` and `yesResult` variable, I just used the `.run` scope function which creates the `this` object and then it falls out of scope at the end of the block.